sketch/RFUT_Elemental.hpp

```diff
@@ -2,6 +2,7 @@
 #define SKYLARK_RFUT_ELEMENTAL_HPP
 
 #include <boost/mpi.hpp>
+#include "spi/include/kernel/memory.h"
```

Comment:
That looks like a BGQ only header. Make sure the code compiles on other architectures as well.